eagles-food-backend/Domains/DTOs/UserDTO.cs

```diff
@@ -4,10 +4,13 @@ namespace eagles_food_backend.Domains.DTOs
 {
     public class CreateUserDTO
     {
+        [RegularExpression(@"^[a-zA-Z- ]*$", ErrorMessage ="Cannot contain special character(s)")]
         public string LastName { get; set; } = null!;
+        [RegularExpression(@"^[a-zA-Z- ]*$", ErrorMessage ="Cannot contain special character(s)")]
         public string FirstName { get; set; } = null!;
         public string Email { get; set; } = null!;
         public string Password { get; set; } = null!;
+        [RegularExpression(@"^[+]?[0789]?[0-9]{10}$", ErrorMessage = "Not a valid phonenumber")]
```

Comment:
not really robust enough, number formats are a pain in the ass
let's just check for a plus and as long as it's all numbers

Reply:
Alright would remove excess checks